includes/class-content-distribution.php

```diff
@@ -172,6 +174,24 @@ public static function handle_post_updated( $post ) {
 		self::$queued_post_updates[] = $post->ID;
 	}
 
+	/**
+	 * Post delete listener callback.
+	 *
+	 * @param int $post_id The post ID.
+	 *
+	 * @return array|null The post payload or null if the post is not distributed.
```

Comment:
This will never return array – should it return or can we fix the comment?

Reply:
Whoops, that's reminiscent from the previous approach. Thank you for catching that. Updated in 647d24710b2fcf595b29220cc9de093848b2eb83